test/help_test.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+/* global describe, beforeEach, afterEach, it, context */
+
+const path = require('path')
+
+const chai = require('chai')
+const expect = chai.expect
+
+chai.use(require('sinon-chai'))
+
+const Hubot = require('hubot')
+const Robot = Hubot.Robot
+const TextMessage = Hubot.TextMessage
+
+const newTestRobot = function newTestRobot () {
+  process.env.PORT = '0'
+  const robot = new Robot(null, 'mock-adapter-v3', true, 'hubot')
+
+  robot.loadFile(path.resolve('src/'), 'help')
+
+  robot.adapter.on('connected', () => robot.brain.userForId('1', {
+    name: 'john',
+    real_name: 'John Doe',
+    room: '#test'
+  }))
+
+  return robot
+}
+
+describe('help', () => describe('getHelpCommands', function () {
```

Comment:
You use arrow function in some places and regular functions in other places, maybe just use arrow functions everywhere?